src/utils/data.ts

```diff
@@ -0,0 +1,53 @@
+import { StaticImageData } from "next/image";
+import chatbtc from "/public/images/chat-btc.jpg";
+import bitcoinSearch from "/public/images/bitcoin-search.jpg";
+import bitcointldr from "/public/images/bitcoin-tldr.jpg";
+import savingSatoshi from "/public/images/saving-satoshi.jpg";
+import transcriptsreview from "/public/images/bitcoin-transcripts-review.jpg";
+import bitcoindevs from "/public/images/bitcoin-devs.jpg";
+
+export type MenuApp = {
+  href: string;
+  image: string | StaticImageData;
+  alt: string;
+  title: string;
+};
+
+export const menuApps = [
+  {
+    href: "https://bitcoindevs.xyz/",
+    image: bitcoindevs,
+    alt: "Bitcoin Devs",
+    title: "Build the future of money - Study & contribute to bitcoin and lightning open source",
+  },
+  {
+    href: "https://chat.bitcoinsearch.xyz",
+    image: chatbtc,
+    alt: "ChatBTC image",
+    title: "Interactive AI chat to learn about bitcoin technology and its history",
+  },
+  {
+    href: "https://bitcoinsearch.xyz/",
+    image: bitcoinSearch,
+    alt: "Bitcoin search",
+    title: "Technical bitcoin search engine",
+  },
+  {
+    href: "https://tldr.bitcoinsearch.xyz/",
+    image: bitcointldr,
+    alt: "Bitcoin TLDR",
+    title: "Daily summary of key bitcoin tech development discussions and updates",
+  },
+  {
+    href: "https://savingsatoshi.com",
+    image: savingSatoshi,
+    alt: "Saving Satoshi",
+    title: "Engaging bitcoin dev intro for coders using technical texts and code challenges",
+  },
+  {
+    href: "https://review.btctranscripts.com/",
+    image: transcriptsreview,
+    alt: "Bitcoin Transcripts Review",
+    title: "Review technical bitcoin transcripts and earn sats",
+  },
```

Comment:
@Emmanuel-Develops, does it make sense to store this information, along with the logos for each product, in the components library as a central source of truth? We're currently repeating them across various products, and in some cases, the copy is even inconsistent.

Reply:
Yes they should be in the component library. Given that the library should be lightweight it would not be wise to have the images in there, rather they should point to where they are hosted. Ideally somewhere on cloudflare.

Cloudflare has a limited free image hosting and that would serve our purposes for now

Reply:
I was mostly thinking about the text data, but your point regarding the images is spot on! I just opened an issue in order to keep track of this https://github.com/bitcoin-dev-project/bdp-ui/issues/30